Lib/test/test_email/test_utils.py

```diff
@@ -147,7 +147,7 @@ def test_localtime_epoch_notz_daylight_false(self):
     def test_variable_tzname(self):
         t0 = datetime.datetime(1984, 1, 1, tzinfo=datetime.timezone.utc)
         t1 = utils.localtime(t0)
-        if t1.tzname() == 'Europe':
+        if t1.tzname() != 'MSK':
```

Comment:
Does not it make the test useless?

We can narrow the skip condition:
```suggestion
        if t1.tzname() in ('Europe', 'UTC'):
```